app/models/get-comments-model.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../../db/connection");
+
+exports.fetchComments = (review_id) => {
+  const commentPromise = db.query(
+    `SELECT * FROM comments WHERE review_id = $1;`,
+    [review_id]
+  );
+
+  const reviewPromise = db.query(
+    `SELECT * FROM reviews WHERE review_id = $1;`,
+    [review_id]
+  );
+
+  return Promise.all([commentPromise, reviewPromise]).then(
+    ([commentResults, reviewResults]) => {
+      if (!reviewResults.rows.length && !commentResults.rows.length) {
```

Comment:
Nothing wrong with this here but i could argue we only need to check the review results to assert that there is no review. If there is no review, there is definitely no comments